build.cmd

```diff
@@ -24,7 +25,7 @@ if defined VisualStudioVersion goto :Run
 
 set _VSWHERE="%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe"
 if exist %_VSWHERE% (
-  for /f "usebackq tokens=*" %%i in (`%_VSWHERE% -latest -property installationPath`) do set _VSCOMNTOOLS=%%i\Common7\Tools
+  for /f "usebackq tokens=*" %%i in (`%_VSWHERE% -latest -prerelease -property installationPath`) do set _VSCOMNTOOLS=%%i\Common7\Tools
```

Comment:
I would prefer we not pass "prerelease" here as I want most folks to stay on the released builds of VS. If folks want to use a prerelease then they should simply start the build from a prerelease command prompt already.

Reply:
Just read your issue. Are you suggesting that there is no released version of the tools that will allow arm to build?

Reply:
> I would prefer we not pass "prerelease" here as I want most folks to stay on the released builds of VS.

If you have installed prerelease version of VS on your machine, you may want to use it for everything. It is certainly true for folks on our team. We do want to hear about issues building with prerelease builds of VS early. The best way to achieve that is to use the prerelease build by default.

We are passing the prerelease switch here in CoreRT repo for this reason.

It matches what you get on other platforms: if you install pre-release build of XCode on Mac, we will pick it up for the build.

Reply:
Fair enough, perhaps I'm trying to be too overprotective :) Lets do it and get feedback on any issues folks hit.

Reply:
> they should simply start the build from a prerelease command prompt already

Problem with that approach is that starting build from prerelease command prompt will always select latest RTM installed. It is caused by prefixing vswhere with the path to standard tool which will not choose prerelease tools unless explicitly instructed to do so.

Reply:
>Are you suggesting that there is no released version of the tools that will allow arm to build

No, but like in my case it is possible to install VS 15 RTM without arm tools.

Reply:
>No, but like in my case it is possible to install VS 15 RTM without arm tools and then it is impossible to build for arm target since build script actively prevents using prerelease VS.

While I'm fine updating to search for prerelease as well but I'm still not sure I understand the connection of VS without arm tools and prerelease versions of VS.  In either case whatever version of VS you have installed needs to have the arm tools in order to build for arm.

Reply:
VS 15 RTM without arm tools when installed side by side with preview VS 15 with arm tools will always prevent using VS 15 arm tools by hijacking detection script in build.cmd and always choosing VS 15 RTM even when run from VS preview developer command prompt.